pkg/abi/linux/ioctl.go

```diff
@@ -170,3 +171,9 @@ const (
 	KCOV_MODE_TRACE_PC  = 2
 	KCOV_MODE_TRACE_CMP = 3
 )
+
+// Constants taken from net/tun.c
+const (
+	MIN_MTU = 68
+	MAX_MTU = 65535
+)
```

Comment:
nit: We should probably make sure files have new lines at the end of them

Reply:
Fixed.